app/posts/ecf-v2/arrivals-transferring-in-an-ect.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Arrivals: transferring in an ECT and collecting programme choices'
+description: "How we've designed to transfer in an ECT from another school and asking for the ECT's programme choices."
+date: 2025-02-28
+---
+
+## Why we’re doing this
+We've designed a user journey which allows for schools to register ECTs at their school that have previously been registered at another school.
+
+We've tried to keep the journey as similar as possible to registering a new ECT. This is to keep the journeys as streamlined as possible, and for this reason we'll only have one call to action for a school to register an ECT.
```

Comment:
Is this true with some of the homepage designs?

Is it more that we wanted to not expect schools to have to understand if an ECT had been registered at a previous school? That it would be too confusing/lead to schools getting it wrong?